plugins/pdr_deterministic_plugin/.ci/run_the_script_there.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -x
+pid=$(ps -fade | grep simulation_telemetry | egrep -v grep | awk '// { print $2 }')
+if [[ ! -z $pid ]]; then kill -9 $pid;fi
```

Comment:
are you the only one that run the sim?
it can kill someone else's sim.

Reply:
this simulation_telemetry is a script that was written here, I will write it with full name, so one cannot be confused, and only one ci at a time can occur.

Reply:
good!